clickhouse-client/src/main/java/com/clickhouse/client/config/ClickHouseClientOption.java

```diff
@@ -372,6 +372,25 @@ public enum ClickHouseClientOption implements ClickHouseOption {
      */
     USE_NO_PROXY("use_no_proxy", false,
             "Whether to access ClickHouse server directly without using system wide proxy including the one defined in JVM system properties."),
+
+    /**
+     *  Whether to access ClickHouse server via proxy.
+     *  Proxy type: IGNORE,
+     *              DIRECT,
+     *              HTTP - if selecting HTTP we need also provide proxy hostname and port.
+     */
+    PROXY_TYPE("proxy_type", "IGNORE",
```

Comment:
You can use enum as the default value instead of String, for instance:
```java
SSL_MODE("sslmode", ClickHouseSslMode.STRICT, "verify or not certificate: none (don't verify), strict (verify)"),
```